lib/db/lowlevel.go

```diff
@@ -604,6 +604,83 @@ func (db *Lowlevel) gcBlocks() error {
 	return db.Compact()
 }
 
+// repairSequence makes sure the sequence numbers in the sequence keys match
+// those in the corresponding file entries. It returns the amount of fixed entries.
+func (db *Lowlevel) repairSequence(folders []string) (int, error) {
+	t, err := db.newReadWriteTransaction()
```

Comment:
I think this should hold the gc lock as well. Probably won't cause issues now, but it rewrites FileInfos and gets called at random times so in principle they are in the same territory.

Reply:
Done.